src/main/java/com/jfrog/ide/idea/ui/JFrogLocalToolWindow.java

```diff
@@ -1,104 +1,175 @@
     @Override
     public JPanel createActionToolbar() {
-        JPanel toolbarPanel = createComponentsTreePanel(true);
-        // Add export menu
-        ExportMenu exportMenu = new ExportMenu(project);
-        componentsTree.addFilterMenu(exportMenu);
-        toolbarPanel.add(exportMenu.getMenuButton());
-        return toolbarPanel;
+        DefaultActionGroup actionGroup = new DefaultActionGroup(new CollapseAllAction(componentsTree), new ExpandAllAction(componentsTree));
+        actionGroup.addAction(ActionManager.getInstance().getAction("JFrog.RefreshLocal"), Constraints.FIRST);
+        return createJFrogToolbar(actionGroup);
     }
 
-    @SuppressWarnings("DialogTitleCapitalization")
-    @Override
-    JComponent createMoreInfoView(boolean supported) {
-        if (!GlobalSettings.getInstance().areXrayCredentialsSet()) {
-            return ComponentUtils.createNoCredentialsView();
+    /**
+     * Register the issues tree listeners.
+     */
+    public void registerListeners() {
+        // Xray credentials were set listener
+        appBusConnection.subscribe(ApplicationEvents.ON_CONFIGURATION_DETAILS_CHANGE, () ->
+                ApplicationManager.getApplication().invokeLater(this::onConfigurationChange));
+
+        // Component selection listener
+        componentsTree.addTreeSelectionListener(e -> {
+            if (e == null || e.getNewLeadSelectionPath() == null || !(e.getNewLeadSelectionPath().getLastPathComponent() instanceof VulnerabilityOrViolation)) {
+                verticalSplit.setSecondComponent(null);
+                return;
+            }
+
+            selectedIssue = (VulnerabilityOrViolation) e.getNewLeadSelectionPath().getLastPathComponent();
+            updateIssueOrLicenseInWebview(selectedIssue);
+            verticalSplit.setSecondComponent(browser.getComponent());
+        });
+        projectBusConnection.subscribe(ApplicationEvents.ON_SCAN_LOCAL_STARTED, () -> ApplicationManager.getApplication().invokeLater(this::resetViews));
+    }
+
+    private void initVulnerabilityInfoBrowser() {
+        if (!JBCefApp.isSupported()) {
+            Logger.getInstance().error("Could not open the issue details view - JCEF is not supported");
+            return;
         }
-        JLabel title = new JBLabel(" More Info");
-        title.setFont(title.getFont().deriveFont(TITLE_FONT_SIZE));
-
-        moreInfoPanel = new JBPanel<>(new BorderLayout()).withBackground(UIUtil.getTableBackground());
-        String panelText = supported ? ComponentUtils.SELECT_COMPONENT_TEXT : ComponentUtils.UNSUPPORTED_TEXT;
-        moreInfoPanel.add(ComponentUtils.createDisabledTextLabel(panelText), BorderLayout.CENTER);
-        issuesDetailsScroll = ScrollPaneFactory.createScrollPane(moreInfoPanel, ScrollPaneConstants.VERTICAL_SCROLLBAR_ALWAYS, ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
-        return new TitledPane(JSplitPane.VERTICAL_SPLIT, TITLE_LABEL_SIZE, title, issuesDetailsScroll);
+
+        Path tempDirPath;
```

Comment:
Let's delete this directory in `dispose()`

Reply:
3 lines below we call `deleteOnExit()`. Isn't it good enough?